wagtail/wagtailadmin/menu.py

```diff
@@ -104,7 +105,7 @@ def __init__(self, label, menu, **kwargs):
 
     @property
     def media(self):
-        return Media(js=['wagtailadmin/js/submenu.js']) + self.menu.media
+        return Media(js=[static('wagtailadmin/js/submenu.js')]) + self.menu.media
```

Comment:
Actually, ignore me. This was fixed in master on Django, but 1.9.x and below still just prepend `STATIC_URL`. Wrapping the call in `static()` is the correct thing to do. Looking at the templatetag code, it seems that `static` is not used as a template tag, and is there just to provide a nice wrapper around `staticfiles_storage.url()`. The actual `{% static %}` templatetag code even calls the `static()` function instead of calling the full `staticfiles_storage.url()` function.